srcpkgs/at-spi2-core/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'at-spi2-core'
```

Comment:
Related #22172

Reply:
> @mnabid libXevie-devel and looks like even libXest-devel are not used while building at-spi2-core. So they may be removed from makedepends I guess.

`libXtst-devel` already pulls down `libXest-devel` as dep. and `libXevie-devel` doesn't seem to be used. I'm removing both from makedeps.